decred_wallet/Features/Security/PinInputView.swift

```diff
@@ -37,17 +38,20 @@ class PinInputView: UIView {
     }
     
     override func draw(_ frame: CGRect) {
-        self.drawCells(in: frame)
-    }
-    
-    func drawCells(in frame: CGRect) {
+        
         // clear current views
         self.layer.sublayers?.removeAll()
+        
         if self.subviews.count > 0 {
-            self.subviews[0].removeFromSuperview()
+            self.subviews.forEach{ $0.removeFromSuperview() }
```

Comment:
The if statement is no longer needed. This line 46 should also move closer to line 43, as the comment on line 42 applies to this statement as well.

Reply:
Fixed in commit e4174d51aed804a470b8d42f08229756f8993915